indexer/indexer_tests/editable_map_object_test.cpp

```diff
@@ -0,0 +1,114 @@
+#include "testing/testing.hpp"
+
+#include "indexer/editable_map_object.hpp"
+
+using osm::EditableMapObject;
+
+UNIT_TEST(EditableMapObject_SetWebsite)
+{
+  EditableMapObject emo;
+  emo.SetWebsite("https://some.thing.org");
+  TEST_EQUAL(emo.GetWebsite(), "https://some.thing.org", ());
+
+  emo.SetWebsite("http://some.thing.org");
+  TEST_EQUAL(emo.GetWebsite(), "http://some.thing.org", ());
+
+  emo.SetWebsite("some.thing.org");
+  TEST_EQUAL(emo.GetWebsite(), "http://some.thing.org", ());
+
+  emo.SetWebsite("");
+  TEST_EQUAL(emo.GetWebsite(), "", ());
+
+}
+
+UNIT_TEST(EditableMapObject_ValidateBuildingLevels)
+{
+  TEST(EditableMapObject::ValidateBuildingLevels(""), ());
+  TEST(EditableMapObject::ValidateBuildingLevels("7"), ());
+  TEST(EditableMapObject::ValidateBuildingLevels("17"), ());
+  TEST(EditableMapObject::ValidateBuildingLevels("25"), ());
+  TEST(!EditableMapObject::ValidateBuildingLevels("26"), ());
+  TEST(!EditableMapObject::ValidateBuildingLevels("ab"), ());
+  TEST(!EditableMapObject::ValidateBuildingLevels(
+      "2345534564564453645534545345534564564453645"), ());
+}
+
+UNIT_TEST(EditableMapObject_ValidateHouseNumber)
+{
+  TEST(EditableMapObject::ValidateHouseNumber(""), ());
+  TEST(EditableMapObject::ValidateHouseNumber("qwer7ty"), ());
```

Comment:
This validator looks insane. Have you ever meet such housenumbers in the OSM database?